h2o-core/src/main/java/water/init/NetworkInit.java

```diff
@@ -98,7 +98,6 @@ public static void initializeNetworkSockets( ) {
         break;
 
       } catch (Exception e) {
-        e.printStackTrace();
```

Comment:
This seems not related to this change. Just to understand - the purpose is to clean up ? :)

Reply:
yes. else we have annoying stack trace all the time in SW.

Reply:
thanks for explanation!